### ISSUE TITLE
Deserialize XML Error - Get-NotificationAction

### ISSUE THREAD

**Issue by astapelfeld** [lordmilko/PrtgAPI#130]
Some notification actions cannot be retrieved:

```powershell
PS /home/abc>  Get-NotificationAction -Id 300
Get-NotificationAction : An error occurred while attempting to deserialize XML element 'injected_snmptrapspec': cannot assign 'null' to value type 'Int32'.
At line:1 char:2
+  Get-NotificationAction -Id 300
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Get-NotificationAction], XmlDeserializationException
+ FullyQualifiedErrorId : PrtgAPI.XmlDeserializationException,PrtgAPI.PowerShell.Cmdlets.GetNotificationAction



```

**What is the output of `Get-PrtgClient -Diagnostic`?**

```powershell
PSVersion      : 6.2.4
PSEdition      : Core
OS             : Linux 4.15.0-76-generic #86-Ubuntu SMP Fri Jan 17 17:24:28 UTC 2020
PrtgAPIVersion : 0.9.12
Culture        : en-US
CLRVersion     : .NETCoreApp,Version=v2.1
PrtgVersion    : 19.4.53.1912
PrtgLanguage   : english.lng


```

**Comment by lordmilko** [comment]
Hi @astapelfeld,

Under the *Send SNMP Trap Section* under the Notification aciton with ID 300 in the PRTG UI, the *Specific Trap Code* field normally has a default value of 0, however I can see that it's actually possible to change this field to *nothing*, resulting in the error that you are seeing.

Are you able to advise whether you've set this field to blank on purpose?

**Comment by astapelfeld** [comment]
Hi @lordmilko ,
'Send SNMP Trap' is disabled for the notification action id 300. The id 300 is not the only one showing this error, there are lots. I suspect that they all were created a long time ago. ID 300 was created accoring the History tab in 2010. Probably a very old version of PRTG set that parameter to 'nothing' instead of 0?

**Comment by lordmilko** [comment]
If it's a very old version of PRTG that makes sense why they would be null.

I have released a pre-release build of PrtgAPI 0.9.13 that includes a fix for this issue. Can you please follow the [manual installation instructions](https://github.com/lordmilko/PrtgAPI/wiki/Installation#manual) and advise whether this build resolves the issue?

Regards,
lordmilko

**Comment by astapelfeld** [comment]
It's not a very old version now, but I suspect that the object was created a long time ago.
I installed the pre-release and it works now. Thank you very much for your help!
